src/script/components/Avatar/GroupAvatar.tsx

```diff
@@ -52,22 +51,27 @@ export const GroupAvatar: React.FC<GroupAvatarProps> = ({users, className}) => {
         }}
         data-uie-name="group-avatar-box-wrapper"
       >
-        {slicedUsers.map(user => (
-          <div
-            key={user.id}
-            className="group-avatar-box"
-            css={{
-              ...CSS_FLEX_CENTER,
-              ...CSS_SQUARE(14),
-              color: user.accent_color(),
-              flex: '0 0 auto',
-              fontSize: 9,
-              fontWeight: 900,
-            }}
-          >
-            {Array.from(user.initials())[0]}
-          </div>
-        ))}
+        <svg
```

Comment:
Looks like the border color is different on designs:

<img width="619" alt="Screenshot 2025-03-12 at 09 44 16" src="https://github.com/user-attachments/assets/a216d8e1-5d69-49ea-be03-fab96109ff1c" />

Reply:
+ border radius should be 8px, it looks smaller on the screenshot